### ISSUE TITLE
Add support for semi-colon separated text files?

### ISSUE THREAD

**Issue by JamesSample** [altair-viz/jupyterlab_voyager#60]
Not sure whether this belongs here or on the main [Voyager2](https://github.com/vega/voyager) repository?

Is it possible to add support for semi-colon separated files, please? Voyager currently supports comma- and tab- separated files, but it would be great to be able to use semi-colon separated files too (especially in continental Europe, where "," is used a a decimal separator). This would also make Jupyter Lab Voyager more consistent with the formats already supported by *Open with > Table* for text files.

**Comment by Baukebrenninkmeijer** [comment]
Yes, this would be extremely useful. Also because the default excel separator is a semicolon I often find myself working with semicolon separated files.
